blocks/api/test/function-refs.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Function references used so that Chai doesn't complain about functions
+ * not being equal.
+ *
+ * These are methods that should be assigned to a valid block.
+ */
+export const edit = function edit() {
```

Comment:
I think we could choose one or the other of the named variable or function:

```js
export function edit() {
```

```js
export const edit = function() {
// OR: 
//  export const edit = () => 'tacos';
```

The one-liner arrow function could work well as an inline function in each test, even if duplicated. Else I tend to prefer named functions with the function keyword.